JitsiParticipant.ts

```diff
@@ -240,19 +241,19 @@ export default class JitsiParticipant {
     }
 
     /**
-     * @returns {Array.<JitsiTrack>} The list of media tracks for this
+     * @returns {Array.<JitsiTrack|JitsiRemoteTrack>} The list of media tracks for this
```

Comment:
JitsiParticipant encapsulates every other participant except us, so it's only concerned with remote tracks.